### DISCUSSION TITLE
Export vault data with CLI

### DISCUSSION THREAD

**Discussion by S1M8N** [dani-garcia/vaultwarden#2280] [Q&A]
Hello everybody,

I have question... How Can I **export my vault data no encrypted with CLI**

I have find that : https://bitwarden.com/help/export-your-data/

But not working into my container...

If you have any idea or solution I gladly take ;)

Thank you in advanced

**Comment by ghost** [comment]


**Reply by tefracky** [reply]
Did you found a solution?

**Reply by S1M8N** [reply]
I have any solution for this specific situation but I don't take more time for that... My "solution" is backup the database with replication process and I think for security is better. But if you have an idea, put it here :)

edit: by default the bitwarden/cli isn't install. Now I understand why the command bw dosen't work.... Try to install bitwarden/cli and check it :)

**Reply by BlackDex** [reply]
The cli is indented to be installed on a client device, not within the container or host device

**Comment by BlackDex** [comment]
https://bitwarden.com/help/export-your-data/#tab-cli-1QlXqfleMlF6jWT87Dbn2u should just work fine.

Just make sure you login correctly following these instructions https://bitwarden.com/help/cli/#log-in

**Reply by S1M8N** [reply]
the bw command not found in vaultwarden container...

**Reply by kpfleming** [reply]
It doesn't need to be in the VW container, you can install the Bitwarden CLI tools anywhere you like.
